docs/user-guide/persistent-volumes/walkthrough.md

```diff
@@ -36,6 +36,31 @@ NAME      LABELS       CAPACITY      ACCESSMODES   STATUS      CLAIM     REASON
 pv0001    type=local   10737418240   RWO           Available 
 ```
 
+### Access Control
+Storage configured with GID will only allow writing by pods using the same GID.
+Mismatched or missing GIDs will cause `permission denied` errors. Adding a GID
+to a `PersistentVolume` via the `pv.beta.kubernetes.io/gid` annotation allows
```

Comment:
I would phrase this as 'annotating a `PersistentVolume` with a GID'.  The gid isn't really added to the volume, per-se.